### ISSUE TITLE
install the system apps into `fabric8-system` namespace?

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/gofabric8#324]
when installing fabric8 with minikube we tend to get things like kube dashboard or helm installed into the `kube-system` namespace. 

I wonder if we should do the same for system services in fabric8 too? 

e.g. install the following services into the `fabric8-system` namespace:
* fabric8 (console)
* exposecontroller
* configmapcontroller
* any ingress/router related stuff 

I wonder what other system services we should install; maybe nexus and content-repository?

**Comment by rawlingsj** [comment]
Yeah, we already install the ingress stuff in fabric8-system and I've been meaning to move exposecontroller + configmapcontroller there too. Moving the console there makes sense also.  It will also make team development namespaces cleaner too.

**Comment by jstrachan** [comment]
yeah - then we can have basically 2 install steps:
* the system (into fabric8-system)
* a team (in any namespace, just installs the team stuff)

might be nice to default to having the local nexus in each team just point at the global nexus? We may wanna do defaults for things like gogs too maybe?

**Comment by rawlingsj** [comment]
That sounds like a good plan to me.

**Comment by jstrachan** [comment]
once we've got the new fabric8-platform package working, we should probably default to install the fabric8 system into the `fabric8` namespace (lazily creating it if it does not exist) unless the user specifies a different namespace
